code/roundstats.dm

```diff
@@ -0,0 +1,46 @@
+
+//This is for the round end stats system.
+
+/hook/roundend/proc/RoundEnd()
+
+	var/cans_opened = cans_opened_temp
+	var/lights_switched_on = lights_switched_on_temp
+	/*var/trash_piles_searched = trash_piles_searched_temp //Those two are only relevant to :b:irgo
+	var/rare_trash_found = rare_trash_found_temp*/
+	var/turbo_lift_floors_moved = turbo_lift_floors_moved_temp
+	var/lost_limbs_shift = lost_limbs_shift_temp
+	//var/mouse_spawned_shift = mouse_spawned_shift_temp //Virgo
+	var/seed_planted_shift = seed_planted_shift_temp
+	var/step_taken_shift = step_taken_shift_temp
+	//var/number_people_walked_over = number_people_walked_over_temp //Also virgo
+	var/destroyed_research_items = destroyed_research_items_temp
+	var/items_sold_shift = items_sold_shift_temp
+	var/disposals_flush_shift = disposals_flush_shift_temp
+
+	world << "<B>Shift facts!</B>"
```

Comment:
`to_chat()` everywhere in here. It's currently just a macro for `<<` so it should work fine. If it's expanded later, then whoever does that needs to be sure that they have a case for output to large numbers of mobs.

Reply:
Are you sure it would be visible to everyone?

Reply:
There may be a `to_world` macro already. /tg/ has one anyway.